extensions/types/vscode-mssql.d.ts

```diff
@@ -1112,7 +1113,7 @@ declare module 'vscode-mssql' {
 		packageFilePath: string;
 		databaseName: string;
 		upgradeExisting: boolean;
-		sqlCommandVariableValues?: Map<string, string>;
+		sqlCommandVariableValues?: Record<string, string>;
```

Comment:
I copied over the state of the file in main in the vscode-mssql repo so that they're synced up, so there are a few changes like this one made in https://github.com/microsoft/vscode-mssql/pull/17683, that hadn't made over to the ADS copy of the file.

Reply:
Thanks for catching this!